src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -318,14 +318,18 @@ public IActionResult TotalEmployees()
         [Authorize(Policy = PolicyNames.CanEditReport)]
         public IActionResult PostTotalEmployees(bool? hasMinimumEmployeeHeadcount)
         {
-            if (!hasMinimumEmployeeHeadcount.HasValue)
-                return new RedirectResult(Url.Action("TotalEmployeesConfirmationRequired", "Report"));
-
             if (hasMinimumEmployeeHeadcount == false)
                 return new RedirectResult(Url.Action("ReportNotRequired"));
 
             var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
 
+            if (!hasMinimumEmployeeHeadcount.HasValue)
+            {
+                ViewBag.CurrentPeriod = report?.Period ?? _currentPeriod;
+                ModelState.AddModelError("confirm-yes", "Please select did your organisation employ more than 250 people on 31 March 2021 or not");
```

Comment:
you can't hardcode a date in here! Also an invalid date at that as 2021 has long since passed.